external_ml/seviri/predictCPHCOT.py

```diff
@@ -0,0 +1,377 @@
+"""
+    This module contains functions to do neural network COT (CMA) and CPH 
+    predictions based on SEVIRI measurements.
+    
+    - func _all_same(): Helper function. Checks whether all elements in a 
+                      list are equal.
+    - func prepare_input_array(): Bring arrays in the right format for the 
+                                  Neural Network.
+    - func getdata_dummy(): For DWD Servers. Load collocated dummy test data. 
+    - func predict_ANN(): To be called to execute the prediction.
+    
+    First author: Daniel Philipp (DWD)
+    ---------------------------------------------------------------------------
+    2020/07/20, DP: Initial version
+    2020/07/23, DP: Added a neural network driver file containing the
+                    backend, paths and thresholds
+    2020/08/13, DP: Added masking of invalid (space) pixels before prediction.
+                    Only valid pixels are predicted to increase efficiency.
+                    Implemented correct fill value usage.
+    2020/08/18, DP: Implemented fully working uncertainty estimation.
+"""
+
+import neuralnet
+import xarray as xr
+import numpy as np
+import readdriver
+import os
+import time
+import logging
+import helperfuncs as hf
+from definitions import (SREAL_FILL_VALUE, BYTE_FILL_VALUE, SREAL,
+                         IS_CLEAR, IS_CLOUD, IS_WATER, IS_ICE)
+#import tensorflow as tf
+import theano
+
+fmt = '%(levelname)s : %(filename)s : %(message)s'
+logging.basicConfig(level=logging.DEBUG,
+                    format=fmt)
+
+theano.gof.compilelock.set_lock_status(False)
+
+def _prepare_input_arrays(vis006, vis008, nir016, ir039, ir062, ir073, ir087,
+				      ir108, ir120, ir134, lsm, skt, networks):
```

Comment:
Indentation contains mixed tabs and spaces and is also under-intended.

Reply:
Corrected with latest commit 0e2113b.